var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -67,13 +68,35 @@ def install(self, spec, prefix):
         options = [
             '--prefix=%s' % prefix,
             '--enable-optimize',
-            '--enable-tools',
+            '--disable-tools',
+            '--disable-mbconvert',
+            '--disable-hexmodops',
+            '--disable-vtkMOABReader',
+            '--disable-mbsize',
+            '--disable-mbskin',
+            '--disable-mbtagprop',
+            '--disable-mbmem',
+            '--disable-spheredecomp',
+            '--disable-mbsurfplot',
+            '--disable-mbpart',
+            '--disable-dagmc',
+            '--disable-gsets',
+            '--disable-mbmerge',
+            '--disable-mbdepth',
+            '--disable-mbcoupler',
+            '--disable-mcnpmit',
+            '--disable-refiner',
+            '--disable-h5mtools',
+            '--disable-mbcslam',
+            '--disable-mbquality',
+            '--disable-ahf',
+            '--disable-mbumr',
+            '--disable-imesh',
```

Comment:
I agree with disabling things that we don't have Spack packages for. With most build systems, if you don't specify, it will install things if the dependencies are present and won't if they aren't. This results in unreproducible builds: something works fine for one user but not for another. It's better to explicitly enable/disable every variant that behaves this way.